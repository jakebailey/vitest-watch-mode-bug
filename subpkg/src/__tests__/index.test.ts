import { test } from "vitest";
import { execa } from "execa";

const tests = [1, 2, 3, 4];

for (const t of tests) {
  test.concurrent(`test ${t}`, async ({expect}) => {
    const { stdout } = await execa("sh", ["-c", `sleep 1; echo ${t+1}`]);
    expect(stdout).toMatchSnapshot();
  })
}
