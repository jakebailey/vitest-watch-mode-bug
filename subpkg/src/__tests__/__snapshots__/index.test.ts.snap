// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`test 1 1`] = `"2"`;

exports[`test 2 1`] = `"3"`;

exports[`test 3 1`] = `"4"`;

exports[`test 4 1`] = `"5"`;
